lib/fhmongodb.js

```diff
@@ -153,13 +153,19 @@ Database.prototype.tearUp = function (auth) {
           if (interval) {
             clearInterval(interval);
           }
-          self.db = db.db(self.name);
```

Comment:
This is interesting, I think I added this as some authentication scenarios were failing without it, all the unit tests seem to be passing now though. Have you tried running the testing-cloud-app with this updated fh-db?